.changeset/bright-elephants-sparkle.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/create-app': patch
+---
+
+Create a new template for scaffolding apps based on the `app-next` package.
```

Comment:
The reader might not know what `app-next` is. Best explain that this is for the new frontend system instead.